spyder/plugins/completion/kite/status.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+Status widget for Kite completions.
+"""
+
+# Standard library imports
+import logging
+
+# Local imports
+from spyder.config.base import _
+from spyder.utils import icon_manager as ima
+from spyder.widgets.status import BaseTimerStatus
+
+logger = logging.getLogger(__name__)
+
+
+class KiteStatus(BaseTimerStatus):
+    """Status bar widget for Kite completions status."""
+    BASE_TOOLTIP = _("Kite completions status")
+
+    def __init__(self, parent, statusbar, plugin):
+        self.plugin = plugin
+        self.tooltip = self.BASE_TOOLTIP
+        self.open_file_updated = True
+        super(KiteStatus, self).__init__(parent, statusbar,
+                                         icon=ima.get_kite_icon())
+
+    def import_test(self):
+        """Raise ImportError if feature is not supported."""
+        try:
+            from spyder.plugins.completion.kite.utils.status import status
+        except Exception as e:
+            logger.debug('Error while fetching Kite status: {0}'.format(e))
+            raise ImportError
+
+    def get_value(self, filename=None, saved=False):
+        """Return Kite completions state."""
+        from spyder.plugins.completion.kite.utils.status import (
+            status, NOT_INSTALLED, RUNNING)
+        kite_status = status()
+        if kite_status == RUNNING and self.open_file_updated:
+            client_status = self.plugin.get_kite_status()
+            if client_status:
+                kite_status = client_status['short']
+                self.tooltip = client_status['long']
+                if (client_status['status'] == 'ready'
+                        or client_status['status'] == 'unsupported'):
+                    self.open_file_updated = False
+            else:
+                kite_status = 'not reacheable'
+                self.tooltip = self.BASE_TOOLTIP
+                self.open_file_updated = False
+        else:
+            self.tooltip = self.BASE_TOOLTIP
+        text = '𝕜𝕚𝕥𝕖: {}'.format(kite_status)
```

Comment:
If we are showing the Kite icon, we don't need to show `𝕜𝕚𝕥𝕖` logo since it's redundant. This saves a little bit of horizontal space on the status bar as well.